spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationRefresh.java

```diff
@@ -110,6 +113,11 @@ private boolean refreshStores() {
                 if (willRefresh) {
                     // Only one refresh Event needs to be call to update all of the
                     // stores, not one for each.
+                    if (eventDataInfo.equals("*")) {
+                        LOGGER.info("Configuration Refresh event triggered by store modification.");
+                    } else {
+                        LOGGER.info("Configuration Refresh Event triggered by " + eventDataInfo);
+                    }
```

Comment:
Instead of having an if/else condition for log something adding an extra logic what do you think about simplifying this one? 
```suggestion
                    LOGGER.info("Configuration Refresh event triggered by {}.", eventDataInfo);
```